mitmproxy/net/tls.py

```diff
@@ -21,29 +21,21 @@
 
 # redeclared here for strict type checking
 class Method(Enum):
-    # TODO: just SSL attributes once https://github.com/pyca/pyopenssl/pull/985 has landed.
-    TLS_SERVER_METHOD = getattr(SSL, "TLS_SERVER_METHOD", 8)
-    TLS_CLIENT_METHOD = getattr(SSL, "TLS_CLIENT_METHOD", 9)
+    TLS_SERVER_METHOD = getattr(SSL, "TLS_SERVER_METHOD")
+    TLS_CLIENT_METHOD = getattr(SSL, "TLS_CLIENT_METHOD")
 
 
-# TODO: remove once https://github.com/pyca/pyopenssl/pull/985 has landed.
```

Comment:
I think the comment here got moved somehow. Let's keep the version check, but remove the `setdefault` lines below. They shouldn't be necessary.